scripts/local-action.sh

```diff
@@ -0,0 +1 @@
+npx @github/local-action run "./actions/$1" index.ts "./actions/$1/.env"
```

Comment:
does it need to be here instead of inlined in the package json?

Reply:
Not sure why, it does not work